ecology.yaml

```diff
@@ -0,0 +1,525 @@
+---
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+  - name: 'collapse_collections'
+    owner: 'nml'
+    tool_panel_section_label: 'Collection Operations'
+
+  - name: transformation
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: anova
+    owner: lecorguille
+    tool_panel_section_label: 'Statistics'
+
+  - name: heatmap
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: univariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: multivariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: biosigner
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: lcmsmatching
+    owner: prog
+    tool_panel_section_label: 'Annotation'
+
+  - name: add_input_name_as_column
+    owner: mvdbeek
+    tool_panel_section_label: 'Text Manipulation'
+ 
+  - name: regex_replace
+    owner: kellrott
+    tool_panel_section_label: 'Text Manipulation'
+
+  - name: find_in_reference
+    owner: jjohnson
+    tool_panel_section_label: 'Filter and Sort'
+
+  - name: normalize_dataset
+    owner: bebatut
+    tool_panel_section_label: 'Statistics'
+    
+  - name: frogs_2_0_0
+    owner: frogs
+    tool_panel_section_label: 'Find Rapidly OTUs from NGS'
+    
+  - name: frogs_3_1_0
+    owner: frogs
+    tool_panel_section_label: 'Find Rapidly OTUs from NGS'
+
+
+  - name: csv_to_tabular
+    owner: mnhn65mo
+    tool_panel_section_label: 'File Conversion'
+    
+
+  - name: tabular_to_csv
+    owner: mnhn65mo
+    tool_panel_section_label: 'File Conversion'
+
+
+  - name: xls2tabular
+    owner: ylebras
+    tool_panel_section_label: 'File Conversion'
+    
+
+  - name: butterfly_analysis
```

Comment:
Any change to move this into ecology with a review? I assume all those do have conda packages?

Reply:
Sure, we have to move it. We prioritize tools to put into ecology and here, some tools are not already on it so on the old mnhn65mo repository. Here (for butterfly_analysis) there is conda packages as dependencies. I will check all!

Reply:
Removed !